package.json

```diff
@@ -63,7 +63,7 @@
     "es6-promisify": "^5.0.0",
     "express": "^4.13.3",
     "govuk-elements-sass": "^3.1.2",
-    "govuk_template_jinja": "0.23.0",
+    "govuk_template_jinja": "0.23.3",
```

Comment:
yarn.lock changes missing and please either check the template has been updated to match the changes in the template version, or remove the template we use and use the packaged one (not the smallest bit of work)

Reply:
have just pushed the lock file now. will compare the templates

Reply:
they just added some new css and changed some fonts by the looks of it. nothing to do on our side